fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1548,7 +1548,9 @@ private boolean updateCatalogAfterVisible(TransactionState transactionState, Dat
 
     // the write lock of database has been hold
     private boolean updateCatalogAfterVisibleBatch(TransactionStateBatch transactionStateBatch, Database db) {
-        Table table = db.getTable(transactionStateBatch.getTableId());
+        // txns in transactionStateBatch do not contain txn with multi-table,
+        // so we just use the first element of tableIdList
```

Comment:
This comment is misleading and conflicted with the the comment of `TransactionStateBatch.getTableIdList()`.  I think what you want to express is "only single table transactions follow this code path for processing."

Reply:
We can also add an assertiion here:
```
assert transactionStateBatch.getTableIdList().size() == 1
```